docs/howto/communicating_results.md

```diff
@@ -90,17 +40,14 @@ A timely partial warning is better than a complete warning that is too late.
 The basic guidance is that the analyst should communicate all of the vulnerability's possible states, to the best of the analyst's knowledge.
 If the analyst knows nothing, all states are possible.
 For example, [*Utility*](#utility) may be [laborious](#utility), [efficient](#utility), or [super effective](#utility).
-In abbreviated form, write this as `U:LESe`.
-Since a capital letter always indicates a new value, this is unambiguous.
 
-The reason a stakeholder might publish something like `U:LESe` is that it expresses that the analyst thought about [*Utility*](#utility) but does not have anything to communicate.
+The reason a stakeholder might publish a decision point with more than one value is that it expresses that the analyst thought about [*Utility*](#utility) but does not have anything to communicate.
```

Comment:
"more than one value" is not the same as "all values". I think the sentence only makes sense in the context of "all values" ? 
Also possible this sentence is just confusing / not something we would recommend people do, so possible option is to remove the whole discussion of the topic, which more broadly is "how to document an analyst looked but didn't learn anything"

That would probably be a separate issue? So the minimum required to just address this translation here is s/more than one value/all values/

Reply:
took the easy way out in e01f6a5

I'll spawn a separate issue to revisit the _Partial or Incomplete Information_ section in the future